### PR TITLE
:heavy_plus_sign: Add tailwind css

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,9 +41,9 @@
   },
   "scripts": {
     "start": "serve --single build/ --listen $PORT",
-    "dev": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "dev": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```
[the remark this answers is not in the document]
Comment (issa-diallo) [reply package.json:46]: this is doc : https://v2.tailwindcss.com/docs/guides/create-react-app

[the remark this answers is not in the document]
Comment (AndreMiras) [reply package.json:46]: Great put that in the commit message please

File: src/App.tsx [hunk 2]
```diff
@@ -32,7 +32,8 @@ import OneSignalManager from './services/oneSignal/OneSignalManager'
 
 library.add(fas, fab, far)
 
-function App (): JSX.Element {
+// eslint-disable-next-line @typescript-eslint/space-before-function-paren
+function App(): JSX.Element {
```
Comment (AndreMiras) [inline src/App.tsx:36]: Why don't you simply put back the space that was there? Is it incompatible with another linter that we have?

Comment (issa-diallo) [reply src/App.tsx:36]: problem of eslint

**Review by AndreMiras — APPROVED** [review]

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**